src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -888,16 +888,16 @@ public function set_status( $status ) {
 		}
 
 
-		$status = wp_update_post( array(
-			'ID' => $this->id,
-			'post_status' => Tribe__Events__Aggregator__Records::$status->{ $status },
-		) );
+		$status = wp_update_post( [
+			'ID'          => $this->id,
+			'post_status' => Tribe__Events__Aggregator__Records::$status->{$status},
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 970430d023849f1fb856ba8bd6c9cdb9</sub>